File: python/test/test_sequence.py
import ThrustRTC as trtc



darr = trtc.device_vector('int32_t', 10)

trtc.Sequence(darr)
print (darr.to_host())

trtc.Sequence(darr, trtc.DVInt32(1))
print (darr.to_host())

trtc.Sequence(darr, trtc.DVInt32(1), trtc.DVInt32(3))
print (darr.to_host())

File: java/test/test_sequence.java
import java.util.*;
import JThrustRTC.*;

public class test_sequence
{
	public static void main(String[] args) 
	{
		  DVVector vec= new DVVector("int32_t", 10);

		  TRTC.Sequence(vec);
		  System.out.println(Arrays.toString((int[])vec.to_host()));

		  TRTC.Sequence(vec, new DVInt32(1));
		  System.out.println(Arrays.toString((int[])vec.to_host()));
		  
		  TRTC.Sequence(vec, new DVInt32(1), new DVInt32(3));
		  System.out.println(Arrays.toString((int[])vec.to_host()));

	}	

}
